scripts/zones/Quicksand_Caves/IDs.lua

```diff
@@ -40,6 +41,7 @@ zones[xi.zone.QUICKSAND_CAVES] =
         HOMEPOINT_SET                 = 11435, -- Home point set!
         UNITY_WANTED_BATTLE_INTERACT  = 11493, -- Those who have accepted % must pay # Unity accolades to participate. The content for this Wanted battle is #. [Ready to begin?/You do not have the appropriate object set, so your rewards will be limited.]
     },
+
```

Comment:
No need for these additional linebreaks, none of the other ID files have them

Reply:
Oh, I had found one that did, so I thought that was the default.

Reply:
See Qufim Island, the only one near it like that haha. Bad luck.